lib/common/util/modules.js

```diff
@@ -1,34 +1,49 @@
+// @flow
+
 import objectPath from 'object-path'
 
-import { getConfigProperty } from './config'
+import {getConfigProperty} from './config'
+
+import type {Feed} from '../../types'
 
-export const getFeed = (feeds, id) => {
+export function getFeed (feeds: ?Array<Feed>, id: string): ?Feed {
   // console.log(feeds, id)
   // TODO: move use_extension to extension enabled??
-  const useMtc = getConfigProperty('modules.gtfsapi.use_extension') === 'mtc'
+  const useMtc: boolean =
```

Comment:
Ah, flow was not picking this up because this is actually due to an incorrect function return signature in the `getConfigProperty` of `lib/common/util/config`.  Will fix.